### ISSUE TITLE
Exploit API

### ISSUE THREAD

**Issue by noraj** [picatz/shodanz#38]
```ruby
require 'shodanz'

client = Shodanz.client.new(key: 'xxx')

client.exploits_api.search('port: 10050')
# or
client.exploits_api.count('port: 22', page: 1)
# or
client.exploit_count('port: 22')
```

I'm never having an answer, like a forever timeout.
Are you able to make it work?

Directly it works.

```
curl -s 'https://exploits.shodan.io/api/count?query=python&key=xxx' | jq
{
  "matches": [],
  "total": 3263
}
```

I tried with `async-http` 0.50.0 and 0.38.3, `async-io` 1.27.1.

**Comment by noraj** [comment]
I also tried

```ruby
require 'shodanz'

api = Shodanz.api.exploits.new(key: 'redacted')
api.search('port: 10050')
```

**Comment by picatz** [comment]
Yeah, something seems to be causing the async reactor under the hood to be stuck. Trying to debug that now.

**Comment by picatz** [comment]
From what I can tell right now, it's something to do with the exploits API using HTTP/2. 🤔

Maybe I need to close the response body explicitly. Not sure yet.

Example setup to debug async errors with shodanz:

```ruby
require 'shodanz'
require 'pry'

Async.logger.debug!

client = Shodanz.client.new

binding.pry
```

**Comment by picatz** [comment]
Ok, I found the issue. Working on the fix now.

**Comment by picatz** [comment]
Fix will be released in next version coming later today 👍

**Comment by picatz** [comment]
Fixed in [`2.0.2`](https://rubygems.org/gems/shodanz/versions/2.0.2).

**Comment by noraj** [comment]
Awesome. I confirm it's working now.
